From 4af9d80dc783e38a8050c0313d50e8ec70e24461 Mon Sep 17 00:00:00 2001
From: Yannick Fertre <yannick.fertre@foss.st.com>
Date: Wed, 8 Feb 2023 11:45:12 +0100
Subject: [PATCH 407/529] ARM: configs: multi_v7_addons: add display part for
 stm32mp1

Add display part (cec, dsi, ts, ltdc, bridge) to multi_v7 addons
fragment file.

Change-Id: I7a9d760205ff091afd87e5e2bdfce04cbe94d2a5
Signed-off-by: Yannick Fertre <yannick.fertre@foss.st.com>
Reviewed-on: https://gerrit.st.com/c/mpu/oe/st/linux-stm32/+/289852
Reviewed-by: Eric FOURMONT <eric.fourmont-ext@st.com>
ACI: CITOOLS <MDG-smet-aci-reviews@list.st.com>
ACI: CIBUILD <MDG-smet-aci-builds@list.st.com>
Domain-Review: Philippe CORNU <philippe.cornu@foss.st.com>
---
 arch/arm/configs/fragment-02-multiv7_addons.config | 8 ++++++++
 1 file changed, 8 insertions(+)

--- a/arch/arm/configs/fragment-02-multiv7_addons.config
+++ b/arch/arm/configs/fragment-02-multiv7_addons.config
@@ -4,6 +4,7 @@ CONFIG_ARM_CPUIDLE=n
 CONFIG_ARM_SCMI_PROTOCOL=y
 CONFIG_ARM_SMC_MBOX=y
 CONFIG_ARM_STM32_CPUIDLE=y
+CONFIG_BACKLIGHT_GPIO=y
 CONFIG_CHR_DEV_SG=y
 CONFIG_CLKSRC_STM32_LP=y
 CONFIG_COMMON_CLK_SCMI=y
@@ -15,8 +16,14 @@ CONFIG_DEBUG_INFO_DWARF_TOOLCHAIN_DEFAUL
 CONFIG_DEBUG_PREEMPT=y
 CONFIG_DEBUG_SECTION_MISMATCH=y
 CONFIG_DNS_RESOLVER=y
+CONFIG_DRM_I2C_ADV7511=y
+CONFIG_DRM_PANEL_ORISETECH_OTM8009A=y
+CONFIG_DRM_PANEL_RAYDIUM_RM68200=y
 CONFIG_DRM_PANEL_ROCKTECH_HX8394=y
+CONFIG_DRM_SII902X=y
 CONFIG_DRM_SIMPLEDRM=y
+CONFIG_DRM_STM=y
+CONFIG_DRM_STM_DSI=y
 CONFIG_DYNAMIC_DEBUG=y
 CONFIG_FORCE_MAX_ZONEORDER=12
 CONFIG_FUTEX=y
@@ -75,6 +82,7 @@ CONFIG_USELIB=y
 CONFIG_VFP=y
 CONFIG_VIDEO_GC2145=m
 CONFIG_VIDEO_OV5640=m
+CONFIG_VIDEO_STM32_HDMI_CEC=y
 CONFIG_VIDEO_ST_MIPID02=m
 CONFIG_ZBOOT_ROM_BSS=0x0
 CONFIG_ZBOOT_ROM_TEXT=0x0
